### PR TITLE
FSharp.Core 4.3.4

### PR DESCRIPTION
**Pull request by ctaggart**
Set the minimum FSharp.Core to .4.3.4 instead of the implicit 4.5.2. See https://github.com/fsharp/fsharp.github.io/pull/89

### CODE REVIEW

**Comment by enricosada** [comment]
@ctaggart 	that's not enough, you need to also disable implicit include adding these properties

```xml
<DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
<DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
```

File: FsAst.fsproj [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <Compile Include="CreateAst.fs" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.3.4" />
```
Comment (enricosada) [inline FsAst.fsproj:24]: that's not enough, you need also to disable the implicit packagereference

```
<DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
<DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
```

**Review by enricosada — CHANGES_REQUESTED** [review]

File: FsAst.fsproj (hunk 1 above)
Comment (ctaggart) [reply FsAst.fsproj:24]: I'm confused why you would need `DisableImplicitFSharpCoreReference`.

Comment (ctaggart) [reply FsAst.fsproj:24]: Happy to merge a pull request for that. Thanks for the other pull requests! Do you want a new version published soon?

**Pull request merged**